helpers.py

```diff
@@ -116,22 +164,11 @@ def __exit__(self, arg1, arg2, arg3):
 
 
 def new_tester_ssh_connection(setup_test_container):
-    config_hide = Config()
-    config_hide.run.hide = True
     with Connection(
         host="localhost",
         user=setup_test_container.user,
         port=setup_test_container.port,
-        config=config_hide,
-        connect_kwargs={
-            "key_filename": setup_test_container.key_filename,
```

Comment:
We need to preserve this, right? The key filename still needs to be specified somehow.

Reply:
I did run this with the tests in mender-configure-module locally, and since the the commands set StrictHostKeyChecking=no, I figured the key was unnecessary. But I can of course add it back if you think we need it.

Reply:
CI approves, https://gitlab.com/Northern.tech/Mender/mender-configure-module/-/pipelines/1271816938, so it doesn't seem like the keys are used

Reply:
Excellent. Good observation and great that we verified it locally and CI.